src/api/operations/externalFormSubmission.queries.graphql

```diff
@@ -0,0 +1,17 @@
+query projectExternalFormSubmissions(
```

Comment:
Suggestion -- what do you think of renaming this to `GetProjectExternalFormSubmissions` for consistency with other graphql queries?

Reply:
sounds good, I adjusted the query name